client/packages/system/src/Sync/api/operations.generated.ts

```diff
@@ -1,7 +1,7 @@
 import * as Types from '@openmsupply-client/common';
 
 import { GraphQLClient } from 'graphql-request';
-import * as Dom from 'graphql-request/src/types.dom';
+import * as Dom from 'graphql-request/dist/types.dom';
```

Comment:
I regenerated, and this is the result.

Reply:
Btw. this works for me but when I generate the api here I still get invalid code that doesn't compile...

Reply:
thanks! yes I was getting some errors introduced. now getting different ones - the fragment path in request requisitions and the extra bit at the bottom of the stocktakes. I thought to make an issue for those after this one.